lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -170,11 +171,10 @@ function checkObjectAcls(bucket, objectMD, requestType, canonicalID) {
 
 function _checkActions(requestType, actions, log) {
     // if requestType isn't in list of controlled actions
-    if (!Object.values(actionMap).includes(requestType)) {
+    if (!Object.keys(actionMap).includes(requestType)) {
         return true;
     }
-    const mappedAction = Object.keys(actionMap)
-        [Object.values(actionMap).indexOf(requestType)];
+    const mappedAction = actionMap[requestType];
```

Comment:
Now this can become even simpler (and more efficient):

```
    const mappedAction = actionMap[requestType];
    if (!mappedAction) {
        return true;
    }
```